### PR TITLE
PSR 2

### PR DESCRIPTION
**Pull request by adam-paterson**

### CODE REVIEW

**Review by dmanners — CHANGES_REQUESTED** [review]
Looks good, seems to have conflicts I think Marius did some clean up earlier today for a lot of these issues but I love the composer scripts. Please rebase and fix the conflicts and we would happy to have this PR. Thanks

**Comment by adam-paterson** [comment]
@dmanners No problem, need to fix the latest commit anyway. I thought I had rebased.

File: composer.json [hunk 1]
```diff
@@ -18,12 +18,17 @@
         "symfony/console": "^3.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "*"
+        "phpunit/phpunit": "*",
+        "squizlabs/php_codesniffer": "^2.8"
```
Comment (sandermangel) [inline composer.json:22]: @adam-paterson awesome! ever checked out https://github.com/phpro/grumphp? they offer a nice integration with pre-commit hooks

Comment (adam-paterson) [reply composer.json:22]: @sandermangel Sure have https://github.com/phpro/grumphp/commits?author=adam-paterson.

Once I've finished my docker build for the project I'll look at a PR for adding Grump

**Comment by adam-paterson** [comment]
@dmanners Conflicts resolved.

**Review by dmanners — APPROVED** [review]
Love it! Thanks man.

File: composer.json (hunk 1 above)
Comment (sandermangel) [reply composer.json:22]: @adam-paterson ah brilliant, big fan of them, use it in most projects. Really improves overall quality

**Pull request merged**